lib/x11utils.pm

```diff
@@ -237,6 +237,11 @@ sub handle_login {
         type_password($mypwd);
         send_key 'ret';
         wait_still_screen;
+        # If there are additional polkit windows, use `esc` to discard them.
+        if (check_screen([qw(authentication-required-user-settings authentication-required-modify-system)], 10)) {
```

Comment:
Please use a multi-tag `assert_screen` with `match_has_tag` instead of `check_screen` with non-zero timeout to prevent introducing any timing dependant behaviour, to save test execution time as well as state more explicitly from the testers point of view what are the expected alternatives. For example:

```perl
assert_screen([qw(yast2_console-finished yast2_missing_package)]);
if (match_has_tag('yast2_missing_package')) {
    send_key 'alt-o';  # confirm package installation
    assert_screen 'yast2_console-finished';
}
```

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details